<?xml version="1.0" encoding="utf-8"?>
<report
	progress="">
	<activity
		result="Failed"
		duration="52.31s"
		durationms="52306"
		type="root"
		rid="e4b1e559bc7c3de"
		totalerrorcount="1"
		totalwarningcount="0"
		totalsuccesstestcasecount="0"
		totalfailedtestcasecount="0"
		totalblockedtestcasecount="0"
		childsuccesscount="0"
		childfailedcount="0"
		childblockedcount="0"
		totalsuccesscount="0"
		totalfailedcount="0"
		totalblockedcount="0"
		totalmaintenancemodecount="0"
		user="Z440"
		rxversion="8.0.0.0"
		host="Z440-PC"
		osversion="Windows 7 Service Pack 1 64bit"
		runtimeversion="4.0.30319.42000"
		procarch="32bit"
		oslanguage="en-US"
		dtnlanguage="ko-KR"
		screenresolution="1920x1080"
		timestamp="2021-02-19 오전 9:36:39"
		timestampiso="2021-02-19T00:36:39"
		endtime="2021-02-19 오전 9:37:31"
		timeoutfactor="1"
		module="NormalJob">
		<detail>
			NormalJob
		</detail>
		<varbindings>
		</varbindings>
		<item
			timeRelativeToTestSuiteStartTime="00:52.306"
			timeRelativeToTestModuleStartTime="00:52.307"
			timeWallClock="09:37:31"
			time="00:52.306"
			level="Error"
			category="Module">
			<message>
				The test run has been aborted by the user.
			</message>
			<metainfo
				loglvl="Error">
			</metainfo>
		</item>
	</activity>
</report>
